### ISSUE TITLE
sync botbr credentials

### ISSUE THREAD

**Issue by langel** [langel/botb_irc_bot#11]
requires registered nick
works in private message

command botbrname (botbr name)
command botbrpass (botbr password)
command synclogin

the bot then recognizes the botbr for hosting files or something or other

**Comment by ianremsen** [comment]
Works only in private message, if done in public, bot calls you a dumbass

**Comment by langel** [comment]
I had some more thinks on this.  Maybe instead of three commands it could be one --
`!identify (botbr name) (botbr pass)`
In regards to having a registered nick; I think if they're currently registered with the server then the bot logs the account sync permanently.  If they are not registered the account sync only lasts until they quit/part #botb (and maybe if the bot restarts).

**Comment by ianremsen** [comment]
If you do it in a public room the bot should call you an idiot.

**Comment by jangler** [comment]
Perhaps this could be done through the site instead? Having people send their non-IRC passwords over IRC is probably not a great practice.

**Comment by langel** [comment]
Yeah, we need some thoughtful architecture here.  Maybe something more like how my roku does it with my plex server.  One side generates a very short alphanumeric hash and the other side verifies.
